cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/MutableSchedulerConfiguration.java

```diff
@@ -112,14 +112,14 @@ public void setServers(int servers) {
     this.servers = servers;
   }
 
-  @JsonProperty("placement_strategy")
-  public String getPlacementStrategy() {
-    return placementStrategy;
+  @JsonProperty("marathon_placement")
```

Comment:
The naming confused me a bit, because AFAIU this is not regarding placing the scheduler using Marathon, but stating a Marathon style placement constraint. Maybe rename this to `marathon_style_placement_constraint` or just `placement_constraint`?